hs_core/hydroshare/utils.py

```diff
@@ -111,6 +111,7 @@ def group_from_id(grp):
     return tgt
 
 
+# TODO: Tastypie left over. This needs to be deleted
```

Comment:
@selimnairb I don't see this being used in the current REST API. If you are seeing its use in hs_core/views/resource_api.py then it is not part of the current REST API. To me all code in resource_api seems dead code - not used anywhere. That needs to be cleaned up as part of the internal api cleanup that I am planning to do soon.